pandas/tests/arrays/test_boolean.py

```diff
@@ -248,7 +248,11 @@ def test_coerce_to_numpy_array():
     tm.assert_numpy_array_equal(result, expected)
     # with missing values will raise error
     arr = pd.array([True, False, None], dtype="boolean")
-    with pytest.raises(ValueError):
+    msg = str(
```

Comment:
don't need the string constructor
```suggestion
    msg = (
```

Reply:
Thanks for the suggestion! I have made the changes.